core/src/main/java/de/evoila/cf/broker/service/impl/DeploymentServiceImpl.java

```diff
@@ -167,6 +167,21 @@ public ServiceInstanceOperationResponse updateServiceInstance(String serviceInst
         return serviceInstanceOperationResponse;
     }
 
+    @Override
+    public ServiceInstanceOperationResponse updateServiceInstanceContext(String serviceInstanceId,
+                                                                         ServiceInstanceUpdateRequest serviceInstanceUpdateRequest)
+            throws ServiceBrokerException, ServiceInstanceDoesNotExistException, ServiceDefinitionDoesNotExistException {
+
+        ServiceInstance serviceInstance = serviceInstanceRepository.getServiceInstance(serviceInstanceId);
+        if (serviceInstance == null) {
```

Comment:
Is this line necessary? The exception throw of the repository should never allow serviceInstance to be empty, doesn't it?

Reply:
Leftover since this method is based on updateUpdateServiceInstance above